src/tosyntax/FStar.ToSyntax.Env.fs

```diff
@@ -1139,20 +1160,22 @@ let prepare_module_or_interface intf admitted env mname (mii:module_inclusion_in
     in
     let namespace_of_module = if List.length mname.ns > 0 then [ (lid_of_ids mname.ns, Open_namespace) ] else [] in
     (* [scope_mods] is a stack, so reverse the order *)
-    let auto_open = List.rev (auto_open @ namespace_of_module) in
+    let auto_open = namespace_of_module @ List.rev auto_open in
```

Comment:
update the comment?

Reply:
No, I think (?) it's correct (it's `scope_mods` that's a stack, not auto_open; but auto_open is used to populate `scope_mods` later in that function).